test/expect_result.jl

```diff
@@ -0,0 +1,81 @@
+using Arrow
+using Test
+
+enumerate_row(value) = enumerate(value)
+enumerate_row(value::String) = [(1, value)]
+
+function generate_arrow(results)
+    arrowed_results = Dict()
+    for result in results
+        key = RAITest.relation_id(result[1], result[2])
+        value = []
+        if isempty(result[2])
+            value = (v1=[],)
+        else
+            # Generate a v column for each column in multi-value rows
+            vs = []
+            for _ in first(result[2])
+                push!(vs, [])
+            end
```

Comment:
```suggestion
            vs = repeat([[]], length(first(result[2])))
```

Reply:
That seems to insert references to the same array. I tried something along the same lines with `[[] for _ in 1:...`